Formula/hof.rb

```diff
@@ -50,10 +50,15 @@ def install
 
   test do
     ENV["HOF_TELEMETRY_DISABLED"] = "1"
-    assert_match "v#{version}", shell_output("#{bin}/hof version")
+    # upstream bug report, https://github.com/hofstadter-io/hof/issues/257
+    # assert_match "v#{version}", shell_output("#{bin}/hof version")
 
-    system bin/"hof", "mod", "init", "cue", "brew.sh/brewtest"
-    assert_predicate testpath/"cue.mods", :exist?
-    assert_equal "module: \"brew.sh/brewtest\"", (testpath/"cue.mod/module.cue").read.chomp
+    system bin/"hof", "mod", "init", "brew.sh/brewtest"
+    assert_predicate testpath/"cue.mod", :exist?
+    expected = <<~EOS
+      module: "brew.sh/brewtest"
+      cue: "0.6.0"
```

Comment:
This could be easily broken when `cue` is bumped to another version. Perhaps it's good enough to only match the first line?

Reply:
you are right, let me improve it.